docs/install-function-mesh.md

```diff
@@ -66,6 +66,8 @@ This example shows how to install Function Mesh through [Helm](https://helm.sh/)
 
    2. Install Function Mesh Operator.
 
+        There are some params for the Function Mesh Operator, and user can customize them for particular purpose.For example, if user want to enable `pprof` for the operator, modify the `pprof.enable` to `true` in values.yaml.
```

Comment:
```suggestion
        There are some parameters for the Function Mesh Operator, and you can customize them for a particular purpose. For example, if you want to enable `pprof` for the operator, set the `pprof.enable` to `true` in the `values.yaml` file.
```
BTW, could you specify which parameters are available? The word "some" here is not clear enough. And, which release does this update belongs to?